assists/subsystem.py

```diff
@@ -264,7 +264,7 @@ def memory_expand( tree, subnode, memory_start = 0xbeef, prop_name = 'memory', v
     # * memory = <address size address size ...>
     # */
     try:
-        mem = [subnode.props( prop_name )[0][0]]
+        mem = json.loads( subnode.props ( prop_name)[0].value )
```

Comment:
We don't want explicit json loads called within this assist. It needs to be at a more generic level.

This is undoing an explicit change that removed all the json.loads() calls, so we can't put them back in.

The LopperProperty can already automatically do the json loads if the pclass is correct, and it can return the len of the json property.

so we should be doing something like:

mem = []
prop = subnode[prop_name]
for i in range(0,len(prop)):
    mem.append( prop[i] )

... and then iterate on m, which are the json loaded chunks.